Neos.Eel/Classes/Helper/StringHelper.php

```diff
@@ -579,7 +579,7 @@ public function rawUrlDecode($string)
      */
     public function htmlSpecialChars($string, $preserveEntities = false)
     {
-        return htmlspecialchars((string)$string, null, null, !$preserveEntities);
+        return htmlspecialchars((string)$string, ENT_COMPAT | ENT_HTML401, 'UTF-8', !$preserveEntities);
```

Comment:
I think we completely overrode the defaults with `null` (=0) beforehand, i.e. ENT_NOQUOTES|ENT_HTML401 (==0)